CMakeLists.txt

```diff
@@ -914,6 +921,44 @@ if(EMSCRIPTEN)
         COMPONENT ktx_js
     )
 
+    add_executable( ktx_js_read interface/js_binding/ktx_wrapper.cpp )
+    target_link_libraries( ktx_js_read ktx_read )
+    target_include_directories(
+        ktx_js_read
+    PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR}/other_include
```

Comment:
Why are other_include and lib needed now. They weren't for the previous read-only build.

Reply:
Because of the new includes of `vk_format_enum.h` and GL headers by `ktx_wrapper` introduced by Phasmatic's changes.

Reply:
Will it help to bracket these includes and any other write-related code in `ktx_wrapper` with`#if KTX_FEATURE_WRITE`?

Reply:
Ah, this reply from you also appears here, just noticed.

Maybe that works, but I'd have to check.

Reply:
Okay, considering that `createFromBuffer` depends on the GL header and in the other comment thread you agreed to keep that for read-only, I don't think we can change this.